community-operators/node-network-operator/node-network-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |
+      [
+        {
+          "apiVersion": "nodenetwork.openshift.io/v1alpha1",
+          "kind": "NodeNetworkConfigurationPolicy",
+          "metadata": {
+            "name": "policy",
+            "labels": {
+              "machineconfiguration.openshift.io/role": "worker"
+            }
+          },
+          "spec": {
+            "priority": 99,
+            "desiredState": {
+              "interfaces": [
+                {
+                  "name": "eth1",
+                  "numVfs": 2,
+                  "mtu": 1200
+                }
+              ]
+            }
+          }
+        }
+      ]
+    certified: "false"
+    containerImage: quay.io/pliurh/node-network-operator:v0.0.1
+    createdAt: 2019/01/31
+    description: An operator to initialize SR-IOV interfaces in Openshift cluster
+    repository: https://github.com/pliurh/node-network-operator
+    support: Red Hat, Inc.
+    capabilities: Basic Install
+    categories: Networking
+  name: node-network-operator.v0.0.1
+  namespace: placeholder
+spec:
```

Comment:
Additional `spec` fields can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#required-fields-within-your-csv